Umbraco.gitignore

```diff
@@ -24,3 +24,8 @@
 
 # Ignore the Models Builder models out of date flag
 **/App_Data/Models/ood.flag
+
+# NEW for version 9 .Net 5 (Core)
+**/umbraco/Data/*
```

Comment:
I would suggest just ignoring the whole folder, but include umbraco/models, making it:

/umbraco/
!/umbraco/models

for the wwwroot, the full umbraco should also be ignore

/wwwroot/umbraco/

Reply:
the full gitignore for umraco 9 would then be:

```
#ignore umbraco backoffice assest from wwwroot
/wwwroot/umbraco/

#ignore umbraco data/views/settings
/umbraco/

#include default location for modelsbuilder output
!/umbraco/models
```

Reply:
Hi @soreng,
you might be right. Lately I realised the _Build > Clean Solution_ command deletes these folders.
But you are missing `**` when the .gitignore is one folder up from project, like it is in a common solution setup for multi project solutions.

```
#ignore umbraco backoffice assest from wwwroot
**/wwwroot/umbraco/

#ignore umbraco data/views/settings
**/umbraco/

#include default location for modelsbuilder output
!**/umbraco/models
```